app/assets/stylesheets/modules/messages.scss

```diff
@@ -23,3 +23,33 @@
     }
   }
 }
+
+.messages {
+  margin-left: 2em;
+
+  h2 {
+    color: $black;
+    float: left;
+    font-size: 1.2em;
+    font-weight: normal;
+  }
+
+  .message {
+    clear: both;
+  }
+
+  .status {
+    color: $cardinal-red;
+  }
+
+  .active {
+    color: $green;
+    font-weight: bold;
+  }
+
+  .edit-message {
+    left: 14px;
+    position: relative;
+    top: 14px;
```

Comment:
This and the floating of the `h2` feels weird to me.  Can we make the `<h2>` `display: inline` instead of floating (which allows us to removing this left/top positioning) and then maybe give one of the surrounding divs a little `bottom-margin` to achieve the spacing we want. Experiment with it and let me know what you think.